docs/deploying-airbyte/local-deployment.md

```diff
@@ -1,65 +1,39 @@
 # Local Deployment
+This page guides you through setting up your Airbyte on your local machine. 
 
-:::info
-These instructions have been tested on MacOS, Windows 10 and Ubuntu 20.04.
+## Prerequisites
+* [Install Docker](https://docs.docker.com/get-docker/) and make sure you have the latest version of `docker-compose` installed.
+* Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/).
```

Comment:
Change "Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/)." to "Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/) and enable Hyper-V Windows Features"